### ISSUE TITLE
Cannot connect - Error dialing trafficcontroller server

### ISSUE THREAD

**Issue by avasseur-pivotal** [cloudfoundry-community/firehose-to-syslog#127]
```
$ cf curl /v2/info
{
   "name": "",
   "build": "",
   "support": "https://support.pivotal.io",
   "version": 0,
   "description": "",
   "authorization_endpoint": "https://login.cf.semea.piv",
   "token_endpoint": "https://uaa.cf.semea.piv",
   "min_cli_version": "6.7.0",
   "min_recommended_cli_version": "6.11.2",
   "api_version": "2.58.0",
   "app_ssh_endpoint": "ssh.cf.semea.piv:2222",
   "app_ssh_host_key_fingerprint": "55:24:0b:0a:e0:fc:2c:33:af:4a:ef:30:a6:71:9f:cb",
   "app_ssh_oauth_client": "ssh-proxy",
   "routing_endpoint": "https://api.cf.semea.piv/routing",
   "logging_endpoint": "wss://loggregator.cf.semea.piv:443",
   "doppler_logging_endpoint": "wss://doppler.cf.semea.piv:443",
   "user": "a45773d9-dd79-4776-bc89-5559260548b6"
}
```

and get this

```
./firehose-to-syslog_darwin_amd64 --api-endpoint https://api.cf.semea.piv --skip-ssl-validation --user=admin --password=XXX --debug
[2016-10-24 15:18:28.004123553 +0200 CEST] Starting firehose-to-syslog 2.1.0
[2016-10-24 15:18:28.782651353 +0200 CEST] Using wss://doppler.cf.semea.piv:443 as doppler endpoint
[2016-10-24 15:18:28.783504431 +0200 CEST] Start filling app/space/org cache.
[2016-10-24 15:18:29.818129671 +0200 CEST] Done filling cache! Found [22] Apps
[2016-10-24 15:18:29.818158872 +0200 CEST] Connected to Fluentd Server! Connecting to Firehose...
[2016-10-24 15:18:29.818228153 +0200 CEST] Exception occurred! Message: Error while reading from the firehose: %v Details: Error dialing trafficcontroller server: malformed ws or wss URL.
Please ask your Cloud Foundry Operator to check the platform configuration (trafficcontroller is ).
[2016-10-24 15:18:29.818241699 +0200 CEST] Exception occurred! Message: Closing connection with traffic controller due to %v Details: Error dialing trafficcontroller server: malformed ws or wss URL.
Please ask your Cloud Foundry Operator to check the platform configuration (trafficcontroller is ).
[2016-10-24 15:18:29.818247502 +0200 CEST] Exception occurred! Message: Failed connecting to Firehose...Please check settings and try again! Details:
```

**Comment by nparekh30** [comment]
I am getting the same error.

**Comment by shinji62** [comment]
I will check.
You can still use this flag `--doppler-endpoint` like  `--doppler-endpoint=wss://doppler.local.pcfdev.io:443`

**Comment by avasseur-pivotal** [comment]
Indeed, using --doppler-endpoint=wss://doppler.cf.semea.piv:443 does work
This must be a bug in parsing /v2/info
Note that I am using PCF Elastic runtime 1.8

**Comment by shinji62** [comment]
Will fix that. 

Envoyé de mon iPhone

**Comment by DaxterM** [comment]
same issue here. Using --dopler-endpoint works on 1.8

**Comment by shinji62** [comment]
Fix with 2.1.2 please repopen if this happen.
